pkg/annotator/apis/config/v1alpha1/types.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	componentbaseconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AnnotatorConfiguration holds configuration for a node annotator.
+type AnnotatorConfiguration struct {
```

Comment:
Would you please keep this config and remove unused other files like scheme, doc.... etc